### PR TITLE
[PORT] Makes HE pipe min temp diff not so low

### PR DESCRIPTION
**Pull request by mystery3525**
## About The Pull Request

Ports: [Citadel-Station-13/Citadel-Station-13#14967](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/14967)

>Most obvious effect is that the space loop will now cool to 2.71 instead of 22.7 kelvins. Less obvious is that heat exchanger pipes are now actually feasible to use for station cooling.
>This is an arbitrary limitation that isn't so much to do with balance as it is to do with atmospherics being "slow". Well, atmospherics is not slow anymore. We can afford to loosen these restrictions.

## Why It's Good For The Game

I can finally efficiently spread fusion heat throughout the hallways via heat exchangers, and maybe cool the SM better idk.

And Ike put it under "[Things to Port](https://github.com/BeeStation/BeeStation-Hornet/projects/2)"

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl: Putnam3145, ported by mystery3525
tweak: HE pipes can now cool to within 1 Kelvin of the environment instead of 20 Kelvin
tweak: HE pipes only require a temp difference of 1 degree to exchange heat, down from 20. This makes HE pipes viable for cooling/heating station rooms.
balance: A knock-on effect of the HE pipe change is that space cooling is much more powerful
/:cl:

### CODE REVIEW

**Comment by Mat05usz** [comment]
Pretty cool, but I'm not really a fan of it. Saving ~90% of SMs is just about reconnecting the stuff to the space loop and then waiting (good engineer knows when it's possible to salvage the SM just by fixing the loop, and when you need to use other measures). With this, and almost 9x faster heat exchange, what's even the point of building multiple layers of space loop? If it works how it's advertised, it may cool shit down faster than it's possible to siphon from the chamber.

**Comment by AnCopper** [comment]
The entire concept of space cooled SM is already stupid because how do you cool something with literally nothing?

**Comment by mystery3525** [comment]
From what I understand it, its actually quite the opposite. You are taking the relatively hotter (loop) gasses and subjecting them to literal outer space (2.7K), where the heat dissipates through heat radiation. If we are going by thermodynamics, a thermomachine-cooled SM is even more dumb as that heat has to go somewhere (the air around the machine) but actually does not. Regardless, I'm doing this PR to allow for building temperature controls throughout the station via HE pipes.

**Comment by williamcll** [comment]
> The entire concept of space cooled SM is already stupid because how do you cool something with literally nothing?

Have you seen how the ISS cools?

**Comment by Lagomorphica** [comment]
Realism arguments are bad, but almost all of what you are saying about realism is also wrong.

Space itself does not make things cold, and the ISS cooling is far more advanced than simply 'radiate heat'. The ISS uses ammonia thats used to take the heat from other areas, then runs it through a total area of almost 156 square meters of metal tubing to dissipate it through radiation. Mylar and klepton exists to stop it OVERHEATING, because direct sunlight causes one side to reach 121 degrees, and one reaches -157 degrees. 

It takes a very, VERY long time for heat to be lost in space, because it can solely be lost in the form of thermal radiation. 7 tons of ammonia is pumped in and out of the ISS radiators non-stop. Additionally, the ISS cooling is extremely efficient - but, their comes an issue when you want to get colder. Going below room temperature gets a lot harder - such as the infrared sensors on satellites, or for this case, the cooling loop of a supermatter. 

Realism is a dumb argument, though, in this case - as much as I don't like Engineering skirts for a similar reason, such as the OOC/IC risk of getting caught in a machine or something (like a conveyor belt), and also because I think making people use jumpsuits in one job is cooler, especially one of the more manual jobs. 

---

Having faster cooling loops is cool though, but could potentially be unbalanced depending on how effectively it works and all. Being 8.4x faster at space loop cooling might be.. eeeehhh, but at the same time, you can achieve the same thing with a fully upgraded freezer in a far more economical practice. The ultimate thing is whether this brings space loop up to standard (takes up more space, easier to get, doesn't need power, can work passively, etc) with the freezer (can be extremely compact and multiple can be added in an array, uses power, can be placed almost anywhere, not passive).

File: code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/atmospherics/pipe/heat_exchanging
 	level = 2
-	var/minimum_temperature_difference = 20
+	var/minimum_temperature_difference = 0.01
```
Comment (ike709) [inline code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm]: IMO having it be `1` would be a more sensible balance between performance and usefulness.

**Review by ike709 — APPROVED** [review]
Still fine though.

File: code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm]: I agree that 1 wuold probably be better

**Review by ike709 — CHANGES_REQUESTED** [review]
Make it `1` (and edit the changelog)

File: code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm (hunk 1 above)
Comment (ike709) [reply code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm]:
```suggestion
	var/minimum_temperature_difference = 1
```

**Comment by mystery3525** [comment]
Updated.

EDIT: Also updated wiki.

**Review by ike709 — APPROVED** [review]

**Pull request merged**